files/default/sql/ddl/3.7.0__initial_tables.sql

```diff
@@ -2473,3 +2505,82 @@ CREATE TABLE IF NOT EXISTS `model_version` (
   CONSTRAINT `model_fk` FOREIGN KEY (`model_id`) REFERENCES `model` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
 ) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
 
+-- FSTORE-612: Feature monitoring
+CREATE TABLE IF NOT EXISTS `monitoring_window_config` (
+    `id` INT(11) NOT NULL AUTO_INCREMENT,
+    `window_config_type` INT(11) NOT NULL,
+    `training_dataset_version` INT(11),
+    `time_offset` VARCHAR(63),
+    `window_length` VARCHAR(63),
```

Comment:
What was the motivation for storing a number in a varchar? Same question for time_offset.

Reply:
To allow for abbreviations such as "1w" (1 week), "2d" (2 days), etc.

Reply:
Possible values are validated with [these regex expressions](https://github.com/javierdlrm/hopsworks-ee/blob/25044d3635cbf0da16a0652d31349d72a9dc7c0c/hopsworks-common/src/main/java/io/hops/hopsworks/common/featurestore/FeaturestoreConstants.java#L171). There are unit tests covering it.